submissions/mustbefail/dom-api-task/script.js

```diff
@@ -0,0 +1,56 @@
+const getMoviesByGenre = async (movieGenre) => {
+  const API_KEY = "6a4ca56d29d657ff19ba2cf05591a088";
+  const imgBaseUrl = "http://image.tmdb.org/t/p/w300/";
+  const reqUrl = `https://api.themoviedb.org/3/discover/movie?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1&primary_release_year=2020&with_genres=${movieGenre}`;
+  const responce = await fetch(reqUrl);
+  const data = await responce.json();
+  const cardsContainer = document.querySelector(".cards-container");
+  cardsContainer.innerHTML = "";
+
+  data.results.forEach((el) => {
+    const movieCard = document.createElement("div");
+    movieCard.classList.add("movie-card");
+    const img = document.createElement("img");
+    const imgUrl = `${imgBaseUrl}${el.poster_path}`;
+    img.setAttribute("src", imgUrl);
+    movieCard.innerHTML = `
+      <img src="${imgUrl}" />
+      <div class="title">
+        <div class="card-header">
+          <h5>${el.title}</h5>
+          <h6>${el.vote_average}</h6>
+        </div>
+        <p class="card-description">
+        ${el.overview.slice(0, 150)}...
+        </p>
+      </div>
+    `;
+    cardsContainer.append(movieCard);
+  });
+  console.log(data);
+};
+
+const handler = (e) => {
+  const { target } = e;
+  const genreId = target.dataset.genre;
+  const currentActive = document.querySelector("a.active");
+  const mainHeader = document.querySelector("main h2");
```

Comment:
Let's query this title by class here. It can cause some misunderstandings when you'll have multiple `h2` on your page. And the same situation with the `currentActive` variable as well.